test/Operators/logicalAndAssignment.js

```diff
@@ -0,0 +1,123 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft Corporation and contributors. All rights reserved.
+// Copyright (c) 2021 ChakraCore Project Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+
+assert.sameValue = function sameValue(expected, actual, message) {
+    assert.areEqual(actual, expected, message);
+}

```

Comment:
What's wrong with the existing `areEqual`? We don't need to look the same as test262....

Reply:
We need some efforts to change from test262. Especially for a early draft PR.😂